jay-config/src/video.rs

```diff
@@ -301,6 +301,14 @@ impl Connector {
     pub fn set_brightness(self, brightness: Option<f64>) {
         get!().connector_set_brightness(self, brightness);
     }
+
+    /// Get both the currently visible workspace and all workspaces on this connector.
+    ///
+    /// If this connector is not connected, or is there no active workspace, returns a
+    /// workspace whose `exists()` returns false.
+    pub fn workspaces(self) -> (Workspace, Vec<Workspace>) {
```

Comment:
I think this should be split into two functions, one that returns the visible workspace and one that returns all workspaces. Same for the IPC messages.